src/styles/blocks/card.scss

```
@@ -0,0 +1,105 @@
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  margin-bottom: 46px;
+  background-color: #fff;
+  color: #060b35;
+  border: 1px solid #f3f3f3;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  
+
+  &:not(:nth-child(4n)) {
+    margin-right: #{$margin-right-card};
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin-bottom: 4px;
+  }
+
+  &__subtitle {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
```

create variables to all colors
```suggestion
    color: $c-someColor;
```